win32_threads.c

```diff
@@ -472,8 +472,15 @@ GC_suspend(GC_thread t)
       }
 
       /* Resume the thread, try to suspend it in a better location.   */
-      if (ResumeThread(t->handle) == (DWORD)-1)
+      if (ResumeThread(t->handle) == (DWORD)-1) {
```

Comment:
If we handle ResumeThread here, I guess we might potentially get same error in GC_start_world, and we might get same error returned by GetThreadContext in GC_push_stack_for.

Reply:
Fixed in c1a45b9ad